### PR TITLE
Sourcery Starbot ⭐ refactored CodingMantras/django-object-detection

### PR DESCRIPTION
**Pull request by SourceryAI**
Thanks for starring [sourcery-ai/sourcery](/sourcery-ai/sourcery) ✨ 🌟 ✨

Here's your pull request refactoring your most popular Python repo.

If you want Sourcery to refactor all your Python repos and incoming pull requests [install our bot](/apps/sourcery-ai/installations/new).

<details>
<summary>Review changes via command line</summary>
<p>To manually merge these changes, make sure you're on the <code>master</code> branch, then run:</p>
<pre>
git fetch https://github.com/sourcery-ai-bot/django-object-detection master
git merge --ff-only FETCH_HEAD
git reset HEAD^
</pre>
</details>

### CODE REVIEW

File: apps/detectobj/views.py [hunk 1]
```diff
@@ -32,9 +32,9 @@ def get_context_data(self, **kwargs):
         # For pagination GET request
         self.get_pagination(context, images_qs)
 
-        is_inf_img = InferencedImage.objects.filter(
-            orig_image=img_qs).exists()
-        if is_inf_img:
+        if is_inf_img := InferencedImage.objects.filter(
+            orig_image=img_qs
+        ).exists():
```
Comment (SourceryAI) [inline apps/detectobj/views.py:37]: Function `InferencedImageDetectionView.get_context_data` refactored with the following changes:

- Use named expression to simplify assignment and conditional ([`use-named-expression`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-named-expression/))

File: apps/detectobj/views.py [hunk 2]
```diff
@@ -48,8 +48,9 @@ def get_pagination(self, context, images_qs):
             images_qs, settings.PAGINATE_DETECTION_IMAGES_NUM)
         page_number = self.request.GET.get('page')
         page_obj = paginator.get_page(page_number)
-        context["is_paginated"] = True if images_qs.count(
-        ) > settings.PAGINATE_DETECTION_IMAGES_NUM else False
+        context["is_paginated"] = (
+            images_qs.count() > settings.PAGINATE_DETECTION_IMAGES_NUM
+        )
```
Comment (SourceryAI) [inline apps/detectobj/views.py:53]: Function `InferencedImageDetectionView.get_pagination` refactored with the following changes:

- Simplify boolean if expression ([`boolean-if-exp-identity`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/boolean-if-exp-identity/))
- Remove unnecessary casts to int, str, float or bool ([`remove-unnecessary-cast`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/remove-unnecessary-cast/))

File: apps/detectobj/views.py [hunk 3]
```diff
@@ -59,10 +60,7 @@ def post(self, request, *args, **kwargs):
 
         # Get form data
         modelconf = self.request.POST.get("confidence")
-        if modelconf:
-            modelconf = float(modelconf)
-        else:
-            modelconf = settings.MODEL_CONFIDENCE
+        modelconf = float(modelconf) if modelconf else settings.MODEL_CONFIDENCE
```
Comment (SourceryAI) [inline apps/detectobj/views.py:63]: Function `InferencedImageDetectionView.post` refactored with the following changes:

- Replace if statement with if expression ([`assign-if-exp`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/assign-if-exp/))

File: apps/images/models.py [hunk 4]
```diff
@@ -29,8 +29,7 @@ def __str__(self):
         return f'{self.name.capitalize()}'
 
     def get_dirpath(self):
-        rootdir = os.path.join(self.user.username, self.name)
-        return rootdir
+        return os.path.join(self.user.username, self.name)
```
Comment (SourceryAI) [inline apps/images/models.py:32]: Function `ImageSet.get_dirpath` refactored with the following changes:

- Inline variable that is immediately returned ([`inline-immediately-returned-variable`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/inline-immediately-returned-variable/))

File: apps/images/views.py [hunk 5]
```diff
@@ -33,17 +33,16 @@ class ImageSetUpdateView(LoginRequiredMixin, UpdateView):
...
-            )
-            context = {
-                'form': form
-            }
-            return render(self.request, 'images/imageset_form.html', context)
+        )
+        context = {
+            'form': form
+        }
+        return render(self.request, 'images/imageset_form.html', context)
```
Comment (SourceryAI) [inline apps/images/views.py:45]: Function `ImageSetUpdateView.form_valid` refactored with the following changes:

- Swap if/else branches [×2] ([`swap-if-else-branches`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/swap-if-else-branches/))
- Remove unnecessary else after guard condition ([`remove-unnecessary-else`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/remove-unnecessary-else/))

File: apps/images/views.py [hunk 6]
```diff
@@ -86,8 +85,10 @@ def post(self, request, *args, **kwargs):
         imageset_id = self.kwargs.get("pk")
         imageset = get_object_or_404(ImageSet, id=imageset_id)
         if self.request.method == 'POST':
-            images = [self.request.FILES.get("file[%d]" % i)
-                      for i in range(0, len(self.request.FILES))]
+            images = [
+                self.request.FILES.get("file[%d]" % i)
+                for i in range(len(self.request.FILES))
+            ]
```
Comment (SourceryAI) [inline apps/images/views.py:91]: Function `ImagesUploadView.post` refactored with the following changes:

- Replace range(0, x) with range(x) ([`remove-zero-from-range`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/remove-zero-from-range/))

File: apps/modelmanager/models.py [hunk 7]
```diff
@@ -106,8 +106,7 @@ def get_classesname(self):
                 return classes.get('names')
         elif self.cls_filetype.lower() == '.txt':
             with open(self.cls_filepath, 'r') as txt_cls_file:
-                classes = txt_cls_file.readlines()
-                return classes
+                return txt_cls_file.readlines()
```
Comment (SourceryAI) [inline apps/modelmanager/models.py:109]: Function `MLModel.get_classesname` refactored with the following changes:

- Inline variable that is immediately returned ([`inline-immediately-returned-variable`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/inline-immediately-returned-variable/))

File: apps/modelmanager/views.py [hunk 8]
```diff
@@ -66,8 +66,7 @@ def get_queryset(self):
         return super().get_queryset().filter(uploader=self.request.user).order_by('-created')
 
     def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        return context
+        return super().get_context_data(**kwargs)
```
Comment (SourceryAI) [inline apps/modelmanager/views.py:69]: Function `UserMLModelListView.get_context_data` refactored with the following changes:

- Inline variable that is immediately returned ([`inline-immediately-returned-variable`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/inline-immediately-returned-variable/))

File: apps/users/views.py [hunk 9]
```diff
@@ -25,9 +25,7 @@ def register(request):
 
 class RegistrationUnderApproval(TemplateView):
     def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        # Do something here!
-        return context
+        return super().get_context_data(**kwargs)
```
Comment (SourceryAI) [inline apps/users/views.py:28]: Function `RegistrationUnderApproval.get_context_data` refactored with the following changes:

- Inline variable that is immediately returned ([`inline-immediately-returned-variable`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/inline-immediately-returned-variable/))

This removes the following comments ( [why?](https://docs.sourcery.ai/Product/FAQ/#why-does-sourcery-sometimes-remove-comments) ):
```
# Do something here!
```

File: apps/users/views.py [hunk 10]
```diff
@@ -40,7 +38,7 @@ def profile(request):
         if u_form.is_valid():
             u_form.save()
             p_form.save()
-            messages.success(request, f'Your account has been updated!')
+            messages.success(request, 'Your account has been updated!')
```
Comment (SourceryAI) [inline apps/users/views.py:41]: Function `profile` refactored with the following changes:

- Replace f-string with no interpolated values with string ([`remove-redundant-fstring`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/remove-redundant-fstring/))

**Comment by rampal-punia** [comment]
Merge Pull from Sourcery Starbot

**Comment by rampal-punia** [comment]
Merge from sourcery-ai-bot

**Review by rampal-punia — APPROVED** [review]

**Comment by rampal-punia** [comment]
Merge commit from Sourcery bot

**Pull request merged**